demos/asp-net-core/postman-collection.json

```diff
@@ -0,0 +1,66 @@
+{
+	"variables": [],
+	"info": {
+		"name": "ASP.NET Core demo - CompositionGateway",
+		"_postman_id": "71510306-c9e1-ad86-ca26-d98bd6fbfad2",
+		"description": "",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Order #1, SalesOnly",
+			"request": {
+				"url": "http://localhost:20295/api/orders/1",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets order 1 details from the Sales endpoint"
+			},
+			"response": []
+		},
+		{
+			"name": "Order #1, ShippingOnly",
+			"request": {
+				"url": "http://localhost:20296/api/shippinginfo/order/1",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets order 1 details from the Shipping endpoint"
+			},
+			"response": []
+		},
+		{
+			"name": "Order #1",
+			"request": {
+				"url": "http://localhost:4457/orders/1",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets order 1 from the composition gateway showing order details composed from different endpoints"
+			},
+			"response": []
+		},
+		{
+			"name": "Orders List, SalesOnly",
+			"request": {
+				"url": "http://localhost:20295/api/orders?pageSize=10&pageIndex=0",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets a list of orders details from the Sales endpoint"
+			},
+			"response": []
+		},
+		{
+			"name": "Orders List",
+			"request": {
+				"url": "http://localhost:4457/orders",
+				"method": "GET",
+				"header": [],
+				"body": {},
+				"description": "gets a list of orders composed with information coming from different endpoits"
+			},
+			"response": []
+		}
+	]
+}
```

Comment:
This file currently contains tabs. Are we OK with that, or would we prefer spaces? (I'd prefer spaces.)

Reply:
It's the way Postman exports it, is it worth to change it to spaces? It's not meant to be used by a  human or manually edited.

Reply:
Ah, in that case let's leave it as tabs.